installer/bundle/bundle_skel_PPC-RHEL.sh

```diff
@@ -0,0 +1,584 @@
+      shift 1
+   done
+
+   if [ "${pkg_list}" = "" ]
+   then
+       # Nothing to update
+       return 0
+   fi
+    echo "----- Updating packages: ($pkg_list) -----"
+
+    [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+    rpm --upgrade $FORCE ${pkg_list}
+}
+
+getInstalledVersion()
+{
+
+    # Parameter: Package to check if installed
+    # Returns: Printable string (version installed or "None")
+    if check_if_pkg_is_installed $1; then
+            local version=`rpm -q $1 2> /dev/null`
+            getVersionNumber $version ${1}-
+    else
+        echo "None"
+    fi
+}
+
+shouldInstall_omi()
+{
+    local versionInstalled=`getInstalledVersion omi`
+    [ "$versionInstalled" = "None" ] && return 0
+    local versionAvailable=`getVersionNumber $OMI_PKG omi-`
+
+    check_version_installable $versionInstalled $versionAvailable
+}
+
+shouldInstall_scx()
+{
+    local versionInstalled=`getInstalledVersion scx`
+    [ "$versionInstalled" = "None" ] && return 0
+    local versionAvailable=`getVersionNumber $OM_PKG scx-`
+
+    check_version_installable $versionInstalled $versionAvailable
+}
+
+#
+# Main script follows
+#
+
+set +e
+
+# Validate package and initialize
+ulinux_detect_installer
+
+while [ $# -ne 0 ]
+do
+    case "$1" in
+        --extract-script)
+            # hidden option, not part of usage
+            # echo "  --extract-script FILE  extract the script to FILE."
+            head -${SCRIPT_LEN} "${SCRIPT}" > "$2"
+            local shouldexit=true
+            shift 2
+            ;;
+
+        --extract-binary)
+            # hidden option, not part of usage
+            # echo "  --extract-binary FILE  extract the binary to FILE."
+            tail -n +${SCRIPT_LEN_PLUS_ONE} "${SCRIPT}" > "$2"
+            local shouldexit=true
+            shift 2
+            ;;
+
+        --extract)
+            verifyNoInstallationOption
+            installMode=E
+            shift 1
+            ;;
+
+        --force)
+            forceFlag=true
+            shift 1
+            ;;
+
+        --install)
+            verifyNoInstallationOption
+            installMode=I
+            shift 1
+            ;;
+
+        --purge)
+            verifyNoInstallationOption
+            installMode=P
+            shouldexit=true
+            shift 1
+            ;;
+
+        --remove)
+            verifyNoInstallationOption
+            installMode=R
+            shouldexit=true
+            shift 1
+            ;;
+
+        --restart-deps)
+            restartDependencies=--restart-deps
+            shift 1
+            ;;
+
+        --source-references)
+            source_references
+            cleanup_and_exit 0
+            ;;
+
+        --upgrade)
+            verifyNoInstallationOption
+            installMode=U
+            shift 1
+            ;;
+
+        --version)
+            echo "Version: `getVersionNumber $OM_PKG scx-`"
+            exit 0
+            ;;
+
+        --version-check)
```

Comment:
@jeffaco It was showing the full name of the file, now showing only the version after correcting the getVersionNumber method.